pcdsdevices/mv_interface.py

```diff
@@ -707,29 +739,35 @@ def movement(scale, direction):
         elif direction == up and len(args) > 1:
             args[1].umvr(scale)
             print("\r {0:4f}".format(args[1].position), end=" ")
-    """
-    Loop takes in user key input and stops when 'q' is pressed
-    """
+
+    # Loop takes in user key input and stops when 'q' is pressed
     is_input = True
     while is_input is True:
-        inp = get_input()
-        if inp == 'q':
+        inp = util.get_input()
+        if inp in ('q', None):
             is_input = False
         else:
             if len(args) > 1 and inp == down:
                 movement(-scale, up)
             elif len(args) > 1 and inp == up:
                 movement(scale, inp)
             elif inp not in(up, down, left, right, shift_down, shift_up):
-                print("\nUp=scale*2, Downw=scale/2,"
-                      "Left=Reverse, Right=Forward\n"
-                      "If more than one motor exits:"
-                      "Up=move y motor up, Down=move y motor down.\n"
-                      " Left=move x motor backwards,"
-                      " Right=move x motor forwards,"
-                      " Shift_Up=scale*2, Shift_down=scale/2\n"
-                      " Press q to quit."
+                print()  # Newline
+                if len(args) == 1:
+                    print(" Left: move x motor backward")
+                    print(" Right: move x motor forward")
+                    print(" Up: scale*2")
+                    print(" Down: scale/2")
+                else:
+                    print(" Left: move x motor left")
+                    print(" Right: move x motor right")
+                    print(" Down: move y motor down")
+                    print(" Up: move y motor up")
+                    print(" Shift_Up: scale*2")
+                    print(" Shift_Down: scale/2")
+                print(" Press q to quit."
                       " Press any other key to display this message.")
+                print()  # Newline
```

Comment:
`'Press any other key ... .\n'`?

Reply:
technically the "correct" way to do this is actually `print(<msg>, end='\n\n')`